### DISCUSSION TITLE
Add form input on another HTML page

### DISCUSSION THREAD

**Discussion by envoys** [YOURLS/YOURLS#3762] [Q&A]
I am sure this is possible and might have already been done before with the API.

Does anyone have an example of having a form on another html page so that users can shorten urls without being on the YOURLS site?

**Comment by dgw** [comment]
Some combination of the [sample public front page](https://github.com/YOURLS/YOURLS/blob/master/sample-public-front-page.txt) and the [sample code to call YOURLS from another server](https://github.com/YOURLS/YOURLS/blob/master/sample-remote-api-call.txt) should do.
